diff -ruN gtk-sharp-1.0.2/sample/rsvg/Makefile.in gtk-sharp-1.0.2-patch/sample/rsvg/Makefile.in
--- gtk-sharp-1.0.2/sample/rsvg/Makefile.in	2004-09-21 20:42:17.000000000 +0200
+++ gtk-sharp-1.0.2-patch/sample/rsvg/Makefile.in	2004-10-30 17:14:23.000000000 +0200
@@ -119,7 +119,7 @@
 am__include = @am__include@
 am__quote = @am__quote@
 install_sh = @install_sh@
-assemblies = ../../glib/glib-sharp.dll ../../pango/pango-sharp.dll ../../atk/atk-sharp.dll ../../gdk/gdk-sharp.dll ../../gtk/gtk-sharp.dll ../../art/art-sharp.dll ../../rsvg/rsvg-sharp.dll
+assemblies = ../../glib/glib-sharp.dll ../../pango/pango-sharp.dll ../../atk/atk-sharp.dll ../../gdk/gdk-sharp.dll ../../art/art-sharp.dll ../../gtk/gtk-sharp.dll ../../rsvg/rsvg-sharp.dll
 references = $(addprefix /r:, $(assemblies))
 
 @ENABLE_RSVG_TRUE@TARGETS = svghelloworld.exe
