diff -ru gcc-3.3.3/gcc/Makefile.in gcc-3.3.3-new/gcc/Makefile.in
--- gcc-3.3.3/gcc/Makefile.in	2004-01-29 05:42:13.000000000 +0100
+++ gcc-3.3.3-new/gcc/Makefile.in	2004-03-20 15:41:54.027682200 +0100
@@ -2386,7 +2386,6 @@
 	(TARGET_MACHINE='$(target)'; srcdir=`cd $(srcdir); ${PWD_COMMAND}`; \
 	SHELL='$(SHELL)' ;\
 	export TARGET_MACHINE srcdir SHELL ; \
-	$(SHELL) ./fixinc.sh `${PWD_COMMAND}`/include $(SYSTEM_HEADER_DIR) $(OTHER_FIXINCLUDES_DIRS); \
 	rm -f include/syslimits.h; \
 	if [ -f include/limits.h ]; then \
 	  mv include/limits.h include/syslimits.h; \
diff -ru gcc-3.3.3/gcc/version.c gcc-3.3.3-new/gcc/version.c
--- gcc-3.3.3/gcc/version.c	2004-02-14 21:20:59.000000000 +0100
+++ gcc-3.3.3-new/gcc/version.c	2004-03-20 15:42:06.787742376 +0100
@@ -6,7 +6,7 @@
    please modify this string to indicate that, e.g. by putting your
    organization's name in parentheses at the end of the string.  */
 
-const char version_string[] = "3.3.3";
+const char version_string[] = "3.3.3 (CRUX)";
 
 /* This is the location of the online document giving instructions for
    reporting bugs.  If you distribute a modified version of GCC,
