diff -ruN gcalctool-4.3.51.orig/Makefile.in gcalctool-4.3.51/Makefile.in
--- gcalctool-4.3.51.orig/Makefile.in	2004-03-31 15:54:41.246880320 -0500
+++ gcalctool-4.3.51/Makefile.in	2004-03-31 15:54:41.380859952 -0500
@@ -648,7 +648,6 @@
 
 #
 # FIXME:
-#  currently scrollkeeper-update does not seem to
 #  remove its database when there is no useful
 #  information contained there. Therefore, with
 #  "make uninstall" you're left with loads of
diff -ruN gcalctool-4.3.51.orig/configure gcalctool-4.3.51/configure
--- gcalctool-4.3.51.orig/configure	2004-03-31 15:54:41.249879864 -0500
+++ gcalctool-4.3.51/configure	2004-03-31 15:54:41.357863448 -0500
@@ -11610,7 +11610,7 @@
     if test "$gt_cv_have_gettext" = "yes" ; then
 
 cat >>confdefs.h <<\_ACEOF
-#define ENABLE_NLS 1
+#define ENABLE_NLS 0
 _ACEOF
 
     fi
diff -ruN gcalctool-4.3.51.orig/help/C/Makefile.in gcalctool-4.3.51/help/C/Makefile.in
--- gcalctool-4.3.51.orig/help/C/Makefile.in	2004-03-31 15:54:41.253879256 -0500
+++ gcalctool-4.3.51/help/C/Makefile.in	2004-03-31 15:54:41.383859496 -0500
@@ -55,7 +55,6 @@
 # generally called from the makefiles for particular formats of documentation.
 #
 # Note that you must configure your package with --localstatedir=/var
-# so that the scrollkeeper-update command below will update the database
 # in the standard scrollkeeper directory.
 #
 # If it is impossible to configure with --localstatedir=/var, then
@@ -426,7 +425,6 @@
 	for file in $(omffile); do \
 		$(INSTALL_DATA) $$file.out $(DESTDIR)$(omf_dest_dir)/$$file; \
 	done
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir) -o $(DESTDIR)$(omf_dest_dir)
 
 uninstall-local-omf:
 	-for file in $(srcdir)/*.omf; do \
@@ -434,7 +432,6 @@
 		rm -f $(omf_dest_dir)/$$basefile; \
 	done
 	-rmdir $(omf_dest_dir)
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir)
 
 clean-local-omf:
 	-for file in $(omffile); do \
diff -ruN gcalctool-4.3.51.orig/help/Makefile.in gcalctool-4.3.51/help/Makefile.in
--- gcalctool-4.3.51.orig/help/Makefile.in	2004-03-31 15:54:41.252879408 -0500
+++ gcalctool-4.3.51/help/Makefile.in	2004-03-31 15:54:41.382859648 -0500

diff -ruN gcalctool-4.3.51.orig/help/de/Makefile.in gcalctool-4.3.51/help/de/Makefile.in
--- gcalctool-4.3.51.orig/help/de/Makefile.in	2004-03-31 15:54:41.254879104 -0500
+++ gcalctool-4.3.51/help/de/Makefile.in	2004-03-31 15:54:41.385859192 -0500
@@ -55,7 +55,6 @@
 # generally called from the makefiles for particular formats of documentation.
 #
 # Note that you must configure your package with --localstatedir=/var
-# so that the scrollkeeper-update command below will update the database
 # in the standard scrollkeeper directory.
 #
 # If it is impossible to configure with --localstatedir=/var, then
@@ -426,7 +425,6 @@
 	for file in $(omffile); do \
 		$(INSTALL_DATA) $$file.out $(DESTDIR)$(omf_dest_dir)/$$file; \
 	done
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir) -o $(DESTDIR)$(omf_dest_dir)
 
 uninstall-local-omf:
 	-for file in $(srcdir)/*.omf; do \
@@ -434,7 +432,6 @@
 		rm -f $(omf_dest_dir)/$$basefile; \
 	done
 	-rmdir $(omf_dest_dir)
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir)
 
 clean-local-omf:
 	-for file in $(omffile); do \
diff -ruN gcalctool-4.3.51.orig/help/es/Makefile.in gcalctool-4.3.51/help/es/Makefile.in
--- gcalctool-4.3.51.orig/help/es/Makefile.in	2004-03-31 15:54:41.255878952 -0500
+++ gcalctool-4.3.51/help/es/Makefile.in	2004-03-31 15:54:41.386859040 -0500
@@ -55,7 +55,6 @@
 # generally called from the makefiles for particular formats of documentation.
 #
 # Note that you must configure your package with --localstatedir=/var
-# so that the scrollkeeper-update command below will update the database
 # in the standard scrollkeeper directory.
 #
 # If it is impossible to configure with --localstatedir=/var, then
@@ -426,7 +425,6 @@
 	for file in $(omffile); do \
 		$(INSTALL_DATA) $$file.out $(DESTDIR)$(omf_dest_dir)/$$file; \
 	done
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir) -o $(DESTDIR)$(omf_dest_dir)
 
 uninstall-local-omf:
 	-for file in $(srcdir)/*.omf; do \
@@ -434,7 +432,6 @@
 		rm -f $(omf_dest_dir)/$$basefile; \
 	done
 	-rmdir $(omf_dest_dir)
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir)
 
 clean-local-omf:
 	-for file in $(omffile); do \
diff -ruN gcalctool-4.3.51.orig/help/fr/Makefile.in gcalctool-4.3.51/help/fr/Makefile.in
--- gcalctool-4.3.51.orig/help/fr/Makefile.in	2004-03-31 15:54:41.256878800 -0500
+++ gcalctool-4.3.51/help/fr/Makefile.in	2004-03-31 15:54:41.387858888 -0500
@@ -55,7 +55,6 @@
 # generally called from the makefiles for particular formats of documentation.
 #
 # Note that you must configure your package with --localstatedir=/var
-# so that the scrollkeeper-update command below will update the database
 # in the standard scrollkeeper directory.
 #
 # If it is impossible to configure with --localstatedir=/var, then
@@ -426,7 +425,6 @@
 	for file in $(omffile); do \
 		$(INSTALL_DATA) $$file.out $(DESTDIR)$(omf_dest_dir)/$$file; \
 	done
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir) -o $(DESTDIR)$(omf_dest_dir)
 
 uninstall-local-omf:
 	-for file in $(srcdir)/*.omf; do \
@@ -434,7 +432,6 @@
 		rm -f $(omf_dest_dir)/$$basefile; \
 	done
 	-rmdir $(omf_dest_dir)
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir)
 
 clean-local-omf:
 	-for file in $(omffile); do \
diff -ruN gcalctool-4.3.51.orig/help/it/Makefile.in gcalctool-4.3.51/help/it/Makefile.in
--- gcalctool-4.3.51.orig/help/it/Makefile.in	2004-03-31 15:54:41.257878648 -0500
+++ gcalctool-4.3.51/help/it/Makefile.in	2004-03-31 15:54:41.388858736 -0500
@@ -55,7 +55,6 @@
 # generally called from the makefiles for particular formats of documentation.
 #
 # Note that you must configure your package with --localstatedir=/var
-# so that the scrollkeeper-update command below will update the database
 # in the standard scrollkeeper directory.
 #
 # If it is impossible to configure with --localstatedir=/var, then
@@ -426,7 +425,6 @@
 	for file in $(omffile); do \
 		$(INSTALL_DATA) $$file.out $(DESTDIR)$(omf_dest_dir)/$$file; \
 	done
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir) -o $(DESTDIR)$(omf_dest_dir)
 
 uninstall-local-omf:
 	-for file in $(srcdir)/*.omf; do \
@@ -434,7 +432,6 @@
 		rm -f $(omf_dest_dir)/$$basefile; \
 	done
 	-rmdir $(omf_dest_dir)
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir)
 
 clean-local-omf:
 	-for file in $(omffile); do \
diff -ruN gcalctool-4.3.51.orig/help/ja/Makefile.in gcalctool-4.3.51/help/ja/Makefile.in
--- gcalctool-4.3.51.orig/help/ja/Makefile.in	2004-03-31 15:54:41.259878344 -0500
+++ gcalctool-4.3.51/help/ja/Makefile.in	2004-03-31 15:54:41.389858584 -0500
@@ -55,7 +55,6 @@
 # generally called from the makefiles for particular formats of documentation.
 #
 # Note that you must configure your package with --localstatedir=/var
-# so that the scrollkeeper-update command below will update the database
 # in the standard scrollkeeper directory.
 #
 # If it is impossible to configure with --localstatedir=/var, then
@@ -426,7 +425,6 @@
 	for file in $(omffile); do \
 		$(INSTALL_DATA) $$file.out $(DESTDIR)$(omf_dest_dir)/$$file; \
 	done
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir) -o $(DESTDIR)$(omf_dest_dir)
 
 uninstall-local-omf:
 	-for file in $(srcdir)/*.omf; do \
@@ -434,7 +432,6 @@
 		rm -f $(omf_dest_dir)/$$basefile; \
 	done
 	-rmdir $(omf_dest_dir)
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir)
 
 clean-local-omf:
 	-for file in $(omffile); do \
diff -ruN gcalctool-4.3.51.orig/help/ko/Makefile.in gcalctool-4.3.51/help/ko/Makefile.in
--- gcalctool-4.3.51.orig/help/ko/Makefile.in	2004-03-31 15:54:41.260878192 -0500
+++ gcalctool-4.3.51/help/ko/Makefile.in	2004-03-31 15:54:41.390858432 -0500
@@ -55,7 +55,6 @@
 # generally called from the makefiles for particular formats of documentation.
 #
 # Note that you must configure your package with --localstatedir=/var
-# so that the scrollkeeper-update command below will update the database
 # in the standard scrollkeeper directory.
 #
 # If it is impossible to configure with --localstatedir=/var, then
@@ -426,7 +425,6 @@
 	for file in $(omffile); do \
 		$(INSTALL_DATA) $$file.out $(DESTDIR)$(omf_dest_dir)/$$file; \
 	done
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir) -o $(DESTDIR)$(omf_dest_dir)
 
 uninstall-local-omf:
 	-for file in $(srcdir)/*.omf; do \
@@ -434,7 +432,6 @@
 		rm -f $(omf_dest_dir)/$$basefile; \
 	done
 	-rmdir $(omf_dest_dir)
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir)
 
 clean-local-omf:
 	-for file in $(omffile); do \
diff -ruN gcalctool-4.3.51.orig/help/sv/Makefile.in gcalctool-4.3.51/help/sv/Makefile.in
--- gcalctool-4.3.51.orig/help/sv/Makefile.in	2004-03-31 15:54:41.261878040 -0500
+++ gcalctool-4.3.51/help/sv/Makefile.in	2004-03-31 15:54:41.392858128 -0500
@@ -55,7 +55,6 @@
 # generally called from the makefiles for particular formats of documentation.
 #
 # Note that you must configure your package with --localstatedir=/var
-# so that the scrollkeeper-update command below will update the database
 # in the standard scrollkeeper directory.
 #
 # If it is impossible to configure with --localstatedir=/var, then
@@ -426,7 +425,6 @@
 	for file in $(omffile); do \
 		$(INSTALL_DATA) $$file.out $(DESTDIR)$(omf_dest_dir)/$$file; \
 	done
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir) -o $(DESTDIR)$(omf_dest_dir)
 
 uninstall-local-omf:
 	-for file in $(srcdir)/*.omf; do \
@@ -434,7 +432,6 @@
 		rm -f $(omf_dest_dir)/$$basefile; \
 	done
 	-rmdir $(omf_dest_dir)
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir)
 
 clean-local-omf:
 	-for file in $(omffile); do \
diff -ruN gcalctool-4.3.51.orig/help/zh_CN/Makefile.in gcalctool-4.3.51/help/zh_CN/Makefile.in
--- gcalctool-4.3.51.orig/help/zh_CN/Makefile.in	2004-03-31 15:54:41.262877888 -0500
+++ gcalctool-4.3.51/help/zh_CN/Makefile.in	2004-03-31 15:54:41.393857976 -0500
@@ -55,7 +55,6 @@
 # generally called from the makefiles for particular formats of documentation.
 #
 # Note that you must configure your package with --localstatedir=/var
-# so that the scrollkeeper-update command below will update the database
 # in the standard scrollkeeper directory.
 #
 # If it is impossible to configure with --localstatedir=/var, then
@@ -426,7 +425,6 @@
 	for file in $(omffile); do \
 		$(INSTALL_DATA) $$file.out $(DESTDIR)$(omf_dest_dir)/$$file; \
 	done
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir) -o $(DESTDIR)$(omf_dest_dir)
 
 uninstall-local-omf:
 	-for file in $(srcdir)/*.omf; do \
@@ -434,7 +432,6 @@
 		rm -f $(omf_dest_dir)/$$basefile; \
 	done
 	-rmdir $(omf_dest_dir)
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir)
 
 clean-local-omf:
 	-for file in $(omffile); do \
diff -ruN gcalctool-4.3.51.orig/help/zh_HK/Makefile.in gcalctool-4.3.51/help/zh_HK/Makefile.in
--- gcalctool-4.3.51.orig/help/zh_HK/Makefile.in	2004-03-31 15:54:41.263877736 -0500
+++ gcalctool-4.3.51/help/zh_HK/Makefile.in	2004-03-31 15:54:41.394857824 -0500
@@ -55,7 +55,6 @@
 # generally called from the makefiles for particular formats of documentation.
 #
 # Note that you must configure your package with --localstatedir=/var
-# so that the scrollkeeper-update command below will update the database
 # in the standard scrollkeeper directory.
 #
 # If it is impossible to configure with --localstatedir=/var, then
@@ -426,7 +425,6 @@
 	for file in $(omffile); do \
 		$(INSTALL_DATA) $$file.out $(DESTDIR)$(omf_dest_dir)/$$file; \
 	done
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir) -o $(DESTDIR)$(omf_dest_dir)
 
 uninstall-local-omf:
 	-for file in $(srcdir)/*.omf; do \
@@ -434,7 +432,6 @@
 		rm -f $(omf_dest_dir)/$$basefile; \
 	done
 	-rmdir $(omf_dest_dir)
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir)
 
 clean-local-omf:
 	-for file in $(omffile); do \
diff -ruN gcalctool-4.3.51.orig/help/zh_TW/Makefile.in gcalctool-4.3.51/help/zh_TW/Makefile.in
--- gcalctool-4.3.51.orig/help/zh_TW/Makefile.in	2004-03-31 15:54:41.264877584 -0500
+++ gcalctool-4.3.51/help/zh_TW/Makefile.in	2004-03-31 15:54:41.395857672 -0500
@@ -55,7 +55,6 @@
 # generally called from the makefiles for particular formats of documentation.
 #
 # Note that you must configure your package with --localstatedir=/var
-# so that the scrollkeeper-update command below will update the database
 # in the standard scrollkeeper directory.
 #
 # If it is impossible to configure with --localstatedir=/var, then
@@ -426,7 +425,6 @@
 	for file in $(omffile); do \
 		$(INSTALL_DATA) $$file.out $(DESTDIR)$(omf_dest_dir)/$$file; \
 	done
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir) -o $(DESTDIR)$(omf_dest_dir)
 
 uninstall-local-omf:
 	-for file in $(srcdir)/*.omf; do \
@@ -434,7 +432,6 @@
 		rm -f $(omf_dest_dir)/$$basefile; \
 	done
 	-rmdir $(omf_dest_dir)
-	-scrollkeeper-update -p $(scrollkeeper_localstate_dir)
 
 clean-local-omf:
 	-for file in $(omffile); do \
