diff -Nru gnome-mime-data-2.4.2/configure.old gnome-mime-data-2.4.2/configure
--- gnome-mime-data-2.4.2/configure.old	2004-11-05 14:36:45.839098848 +0000
+++ gnome-mime-data-2.4.2/configure	2004-11-05 14:37:04.403276664 +0000
@@ -3690,7 +3690,7 @@
 
     fi
   fi
-     USE_NLS=yes
+     USE_NLS=no
 
 
     gt_cv_have_gettext=no
@@ -4747,7 +4747,7 @@
     if test "$gt_cv_have_gettext" = "yes" ; then
 
 cat >>confdefs.h <<\_ACEOF
-#define ENABLE_NLS 1
+#define ENABLE_NLS 0
 _ACEOF
 
     fi
